### PR TITLE
[4.0] Brought checkbox and its label in the same line

### PR DESCRIPTION
**Pull request by saumyasarkar11**
Pull Request to bring label "Remember me" and its checkbox in the same line .

### Actual result BEFORE applying this Pull Request

Screenshot from 2021-03-23 15-25-15

### Expected result AFTER applying this Pull Request

Screenshot from 2021-03-23 15-24-04

### Documentation Changes Required

None

### CODE REVIEW

File: components/com_users/tmpl/login/default_login.php [hunk 1]
```diff
@@ -60,14 +60,12 @@
 
 			<?php if (PluginHelper::isEnabled('system', 'remember')) : ?>
 				<div  class="com-users-login__remember control-group">
-					<div class="control-label">
-						<label for="remember">
+					
+					<input id="remember" type="checkbox" name="remember" class="inputbox" value="yes">
+					
+					<label for="remember" style="margin-left:3px;">
```
Comment (drmenzelit) [inline components/com_users/tmpl/login/default_login.php]: No inline style, please

**Comment by brianteeman** [comment]
dont we have a class form-check-inline for this?

**Comment by drmenzelit** [comment]
https://getbootstrap.com/docs/5.0/forms/checks-radios/#checks

**Comment by saumyasarkar11** [comment]
Working on it

**Comment by saumyasarkar11** [comment]
Please go through the new commit @brianteeman @drmenzelit

File: components/com_users/tmpl/login/default_login.php [hunk 2]
```diff
@@ -60,14 +60,14 @@
...
 				<div  class="com-users-login__remember control-group">
-					<div class="control-label">
-						<label for="remember">
-							<?php echo Text::_('COM_USERS_LOGIN_REMEMBER_ME'); ?>
-						</label>
-					</div>
-					<div class="controls">
-						<input id="remember" type="checkbox" name="remember" class="inputbox" value="yes">
-					</div>
+
```
Comment (brianteeman) [inline components/com_users/tmpl/login/default_login.php]: not 100% certain so please check but you can probably remove control-group

**Comment by richard67** [comment]
@saumyasarkar11 At the bottom of your PR you can see that CI checks by drone are failing:



If you follow the "Details" link you come to a page where you can see the tests at the left hand side. The one called PHPCS failed (red colour). If you then click on that PHPCS you will see the log file:

https://ci.joomla.org/joomla/joomla-cms/41089/1/6

This one then report code style errors:

```
FILE: /********/src/components/com_users/tmpl/login/default_login.php
----------------------------------------------------------------------
FOUND 4 ERRORS AFFECTING 4 LINES
----------------------------------------------------------------------
 65 | ERROR | [x] Tabs must be used to indent lines; spaces are not
    |       |     allowed
    |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
 66 | ERROR | [x] Tabs must be used to indent lines; spaces are not
    |       |     allowed
    |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
 67 | ERROR | [x] Tabs must be used to indent lines; spaces are not
    |       |     allowed
    |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
 68 | ERROR | [x] Tabs must be used to indent lines; spaces are not
    |       |     allowed
    |       |     (Generic.WhiteSpace.DisallowSpaceIndent.SpacesUsed)
----------------------------------------------------------------------
PHPCBF CAN FIX THE 4 MARKED SNIFF VIOLATIONS AUTOMATICALLY
```

If you fix these code style errors and push the changes, the CI checks will start again, and then they should pass.

Code style is important for a project's maintainability.

The PHPCS is important because if that fails, the automated system tests are not run, which would show other possible errors.

**Comment by saumyasarkar11** [comment]
I think I need to go a bit deeper. Working on it

**Comment by richard67** [comment]
Maybe 2 helpful links regarding code style:

- Find an instruction on how to check and fix PHP code with our code style rules for Joomla 4 here: https://docs.joomla.org/Joomla_CodeSniffer
- Find a description of our code style rules also for other languages here for Joomla 3: https://developer.joomla.org/coding-standards/basic-guidelines.html ... these basically also apply to Joomla 4, just that in J4 we have scss and not less.

**Comment by saumyasarkar11** [comment]
@richard67 Seems fine this time

**Comment by drmenzelit** [comment]
Please check the comment from Brian: https://github.com/joomla/joomla-cms/pull/32821#discussion_r599482289

**Comment by saumyasarkar11** [comment]
> Please check the comment from Brian: [#32821 (comment)](https://github.com/joomla/joomla-cms/pull/32821#discussion_r599482289)

I've removed that additional div. He had commented on the outdated version

**Comment by richard67** [comment]
> I've removed that additional div. He had commented on the outdated version

@saumyasarkar11 I'm not sure. I thin Brian meant this one: `<div  class="com-users-login__remember control-group">`

https://github.com/joomla/joomla-cms/pull/32821/files#diff-4b87fc46e3b0b4efde5b14360a34efca58df76b798e0956948ce45a3e755da0fR62

And as far as I understood he was not sure about if it can be removed or not, but it could be.

@brianteeman Please correct me if I understood you wrong.

If possible, could you check that?

**Comment by drmenzelit** [comment]
@saumyasarkar11 it is a simple check: look into the scss/css files from Cassiopeia template and check if the control-group class is still defined.

**Comment by saumyasarkar11** [comment]
> @saumyasarkar11 it is a simple check: look into the scss/css files from Cassiopeia template and check if the control-group class is still defined.

It is still defined but it is as follows:
.control-group {
  margin: 1em 0;
}
May be we can remove it

**Comment by drmenzelit** [comment]
No, the class is still been used in several views, e.g. `/components/com_content/tmpl/form/edit.php`
Related issue: https://github.com/joomla/joomla-cms/issues/31218

**Comment by saumyasarkar11** [comment]
> No, the class is still been used in several views, e.g. `/components/com_content/tmpl/form/edit.php`
> Related issue: #31218

So should I put it back in?

**Comment by drmenzelit** [comment]
Not necessary, your PR looks good now.

**Comment by drmenzelit** [comment]
I have tested this item :white_check_mark: successfully on dd33b423df6a851c93240b734e8f05c1ebf9a426<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32821">issues.joomla.org/tracker/joomla-cms/32821</a>.</sub>

File: components/com_users/tmpl/login/default_login.php [hunk 3]
```diff
@@ -59,15 +59,13 @@
 			<?php endif; ?>
 
 			<?php if (PluginHelper::isEnabled('system', 'remember')) : ?>
-				<div  class="com-users-login__remember control-group">
-					<div class="control-label">
-						<label for="remember">
+				<div  class="com-users-login__remember">
```
Comment (brianteeman) [inline components/com_users/tmpl/login/default_login.php]:
```suggestion
				<div class="com-users-login__remember">					
```

please remove extra space

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on dd33b423df6a851c93240b734e8f05c1ebf9a426<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32821">issues.joomla.org/tracker/joomla-cms/32821</a>.</sub>

**Comment by ChristineWk** [comment]
https://github.com/joomla/joomla-cms/pull/32821#discussion_r599796595
was not seen before (under Issue)

File: components/com_users/tmpl/login/default_login.php (hunk 3 above)
Comment (ChristineWk) [reply components/com_users/tmpl/login/default_login.php]: > 
> 
> please remove extra space

was not seen before (under Issue)

**Comment by saumyasarkar11** [comment]
extra space removed. Please review

> I have tested this item  successfully on [dd33b42](https://github.com/joomla/joomla-cms/commit/dd33b423df6a851c93240b734e8f05c1ebf9a426)
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/32821](https://issues.joomla.org/tracker/joomla-cms/32821).

Extra space removed. Please review

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32821">issues.joomla.org/tracker/joomla-cms/32821</a>.</sub>

**Comment by saumyasarkar11** [comment]
> RTC
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/32821](https://issues.joomla.org/tracker/joomla-cms/32821).

It says Failure: Number of issues exceeds sum threshold (1/0). What does it mean? @Quy

**Comment by richard67** [comment]
> > RTC
> > This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/32821](https://issues.joomla.org/tracker/joomla-cms/32821).
> 

"RTC" means "ready to commit" and is the status which a PR gets here when it has 2 successful human tests by someone else than the author.

> It says Failure: Number of issues exceeds sum threshold (1/0). What does it mean? @Quy

The failure in the "analysis4x" step drone CI isn't related to this PR. It's a security check (using RIPS) which sometimes might produce false alarms until adjusted, clarification is already ongoing.

**Pull request merged**